crates/pop-cli/src/commands/install/mod.rs

```diff
@@ -1,33 +1,344 @@
 // SPDX-License-Identifier: GPL-3.0
-
+use crate::style::{style, Theme};
 use anyhow::Context;
 use clap::Args;
+use cliclack::{clear_screen, confirm, intro, log, outro, set_theme};
+use duct::cmd;
+use os_info::Type;
+use strum::Display;
 use tokio::{fs, process::Command};
 
+#[derive(Display)]
+pub enum DEPENDENCIES {
```

Comment:
Note, only constants should be in upper case.

Reply:
Mistake!